tests/www/test_views.py

```diff
@@ -2601,14 +2601,14 @@ def test_create_dagrun(self):
 
         dr = self.session.query(models.DagRun).one()
 
-        self.assertEqual(dr.execution_date, timezone.convert_to_utc(datetime(2018, 7, 6, 5, 4, 3)))
+        self.assertEqual(dr.execution_date, dt.fromisoformat('2018-07-06 05:04:03-02:00'))
 
     def test_create_dagrun_valid_conf(self):
         conf_value = dict(Valid=True)
         data = {
             "state": "running",
             "dag_id": "example_bash_operator",
-            "execution_date": "2018-07-06 05:05:03",
+            "execution_date": "2018-07-06 05:05:03-02:00",
```

Comment:
Is it still possible to create it without specifying a timezone too?

Reply:
I don't think so. correct me if i am wrong

Reply:
I guess the html form would allow it, but the JS that populates/updates that field doesn't allow it.

This is probably fine.

Reply:
yeah i poked around on the dag run page, seems like bootstrap JS makes sure a timezone is populated which was currently causing the issue